### ISSUE TITLE
one issue about code

### ISSUE THREAD

**Issue by 7tu** [Zirui0623/A2FSeg#4]
Hello, sorry to bother you again. I think the above error may be due to the incorrect json file, which results in the data not being successfully imported and unable to run. I would like to ask if you have done any other operations, because the data processing is according to your instructions on github. The steps are to first perform dataset_conversation and nnUNet_plan_and_preprocess.py -t 82 --verify_dataset_integrity, but the generated json file seems to be wrong. Sorry for taking up your time. I look forward to your reply. Thank you very much.

**Comment by Zirui0623** [comment]
Yes, there are only two parts in the data processing part: dataset_conversation.py and nnUNet_plan_and_preprocess.py. You can refer to the code of the [nnunet](https://github.com/MIC-DKFZ/nnUNet/tree/nnunetv1) project. This project is based on the open source project nnunet, and the default settings of the data processing part are the same as nnunet.
